### PR TITLE
HPCC-28376 Update the docs ECL Command Line Options

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>
Please see the HPCC Systems Client Tools book. 
http://10.224.20.18/view/Docs-gp/job/Doc-Build-04-GPRepo2/16/artifact/build-docs/Release/docs/EN_US/HPCCClientTools_EN_US-8.12.0-rc3.pdf
to view in context
 

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful integration tests produced: http://10.224.20.18/view/Docs-gp/job/Doc-Build-04-GPRepo2/16/artifact/build-docs/Release/docs/EN_US/HPCCClientTools_EN_US-8.12.0-rc3.pdf

### CODE REVIEW

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml [hunk 1]
```diff
@@ -1220,94 +1529,202 @@ ecl run thor findperson.ecl -I C:\MyECL\
...
+                  <entry>-checkDirty</entry>
 
-                  <entry>Do not include the current directory in -I</entry>
+                  <entry>This option reports any modified attributes using git
+                  status</entry>
                 </row>
 
                 <row>
-                  <entry>--fastsyntax</entry>
+                  <entry>-checkDirty</entry>
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:1549]: This option seems to be repeated

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Comments inline

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml (hunk 1 above)
Comment (g-pan) [reply docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:1549]: good catch!

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV, but @afishbeckln should have the last word

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml (hunk 1 above)
Comment (afishbeck) [reply docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:1542]: And the description on first of the two doesn't match other -checkDirty entries.  So the second description should win.

**Review by afishbeck — CHANGES_REQUESTED** [review]
@g-pan just a few comments.

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml [hunk 2]
```diff
@@ -150,7 +150,8 @@
               <row>
                 <entry>zapgen</entry>
 
-                <entry>returns the zap file name from the given WUID.</entry>
+                <entry>generate and download the entire zap file for the given
+                WUID.</entry>
```
Comment (afishbeckln) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:154]: Sorry, my bad, I think I used the word "entire" but now that I see it I think that word should be removed.

**Review by afishbeckln — CHANGES_REQUESTED** [review]
@g-pan one comment where I think I included and extra word I now see shouldn't be included.

**Review by afishbeck — APPROVED** [review]
@g-pan looks good.

**Review by afishbeckln — APPROVED** [review]

**Comment by g-pan** [comment]
Completed the additional change, squashed. Would not let me remove the duplicate reviewer.

**Comment by afishbeck** [comment]
> Completed the additional change, squashed. Would not let me remove the duplicate reviewer.

Yeah, sorry about the duplicate reviewer.  I need to be more careful about mixing my accounts.

**Comment by ghalliday** [comment]
@g-pan please squash

**Pull request merged**